powertools-metrics/src/main/java/software/amazon/lambda/powertools/metrics/MetricsUtils.java

```diff
@@ -137,7 +152,8 @@ private static MetricsLogger logger() {
         MetricsContext metricsContext = new MetricsContext();
 
         if (hasDefaultDimension()) {
-            metricsContext.setDefaultDimensions(defaultDimensionSet());
+            metricsContext.setDefaultDimensions(new DimensionSet());
```

Comment:
I think this is not needed as `setDimensions` overrides the defaults or is there something am missing?

Reply:
That is true.